faq/privacy.php

```diff
@@ -51,51 +52,57 @@
     ),
     array(
         _('How did you hear about us: Your answer to the "How did you hear about us?" question on the registration form is for general statistical purposes and any tie to your profile will be visible only to our site administrators.'),
     ),
     array(
         _('During the various production phases, your User Name and your %s work may be viewable by other volunteers.'),
-        _('In addition, volunteers delegated to an evaluation or mentoring role will be able to review the work performed under your User Name.'),
+        _('In addition, administrators and volunteers delegated to an evaluation or mentoring role will be able to review the work performed under your User Name.'),
     ),
     array(
-        _('You have the option to make your %s statistics (Information normally viewable on the Statistics General page such as the number of pages you have proofread, when you were last on the site, etc.) private so that only registered users can see them, public so that anyone including non-registered visitors can see them, or anonymous so that only you and site administrators can see them.'),
+        _('You have the option to make your %1$s statistics private so that logged-in users can see them, or anonymous so that only you and %1$s administrators can see them.'),
+        _('This includes information such as the number of pages you have proofread, when you were last on the site, etc., that is viewable from your member details (personal stats) page, the sidebar on the round pages, and the Statistics Central page.'),
     ),
     array(
-        _('Your User Name and any public information you provide in your forum profile will be accessible to other %s volunteers and to unregistered guests.'),
+        _('Your User Name and any public information you provide in your forum profile will be accessible to other %s volunteers and to unregistered guests using the forums.'),
         _('Your posts on our discussion forums will be viewable by other volunteers.'),
         _('Certain clearly-designated forums are also viewable by unregistered guests to the forums.'),
-        _('Your User Name and any information you post on the wiki are visible to both volunteers and unregistered guests.'),
+        _('Your User Name and any information you post on the wiki are visible to both volunteers and unregistered guests using the wiki.'),
     )
 );
 
 output_section(
     _('Tracking information'),
     array(
         _('%s tracks your User Name, the date your account was created and your last login date.'),
-        _('This tracking information will be available to other volunteers.'),
+        _('If you set your statistics to "anonymous," the date your account was created and your last login date will be visible only to administrators.'),
```

Comment:
> `"anonymous,"`

I know this is how it's "suppose to be" in American English, but I cringe with the comma being inside the quote, because `anonymous,` is not the setting -- it's `anonymous`. If we aren't willing to put the comma outside, can we sidestep it and use something besides quotes to emphasize the setting name?

Reply:
For what it's worth, I agree with you. That's the way I got it, and since I admitted that it was a legitimate construction (even though it looks wrong to me), I left it.

Reply:
Fixed.